.github/workflows/ci.yaml

```diff
@@ -95,14 +93,6 @@ jobs:
       - name: Test
         run: make test
 
-      - name: Upload coverage reports to Codecov
-        uses: codecov/codecov-action@v4.0.1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          slug: numaproj/numaplane
-          fail_ci_if_error: true
-          verbose: true
-
```

Comment:
No need to upload unit test coverage as we are already uploading merged coverage